src/test/script_tests.cpp

```diff
@@ -40,7 +40,7 @@ using namespace json_spirit;
 using namespace boost::algorithm;
 
 // Uncomment if you want to output updated JSON tests.
-// #define UPDATE_JSON_TESTS
+#define UPDATE_JSON_TESTS
```

Comment:
FWIW, I also noticed this but since I didn't see the generated files in the git repository, I concluded it sense.